armory/__main__.py

```diff
@@ -760,6 +761,10 @@ def main():
     elif sys.argv[1] in ("-v", "--version", "version"):
         print(f"{armory.__version__}")
         sys.exit(0)
+    elif sys.argv[1] == "--show-docker-version-tag":
+        version = armory.utils.version.get_version()
+        print(armory.utils.version.to_docker_tag(version))
```

Comment:
Do we want to make this use the same pattern as `--version`, replacing this with
```
print(armory.utils.version.to_docker_tag(armory.__version__))
```
?

Reply:
https://github.com/twosixlabs/armory/issues/1654